### ISSUE TITLE
 `contextcheck` linter is broken in Github Actions

### ISSUE THREAD

**Issue by ErikPelli** [golangci/golangci-lint#4551]
### Welcome

- [X] Yes, I'm using a binary release within 2 latest major releases. Only such installations are supported.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've read the typecheck section of the FAQ (https://golangci-lint.run/usage/faq/#why-do-you-have-typecheck-errors).
- [X] Yes, I've tried with the standalone linter if available (e.g., gocritic, go vet, etc.). (https://golangci-lint.run/usage/linters/)

### Description of the problem

I was using the linter `contextcheck` with golangci-lint version 1.56.2 and earlier and there was absoloutely no problem,
Yesterday golangci-lint 1.57.0 was released and in the pipelines of our companies the following error appeared:
```
  Running [/home/runner/golangci-lint-1.57.0-linux-amd64/golangci-lint run --out-format=github-actions] in [] ...
  Error: Function `Start$1` should pass the context parameter (contextcheck)
  
  level=warning msg="[runner] Can't process result by autogenerated_exclude processor: can't filter issue result.Issue{FromLinter:\"contextcheck\", Text:\"Function `Start$1` should pass the context parameter\", Severity:\"\", SourceLines:[]string(nil), Replacement:(*result.Replacement)(nil), Pkg:(*packages.Package)(0xc0014d6480), LineRange:(*result.Range)(nil), Pos:token.Position{Filename:\"\", Offset:0, Line:0, Column:0}, HunkPos:0, ExpectNoLint:false, ExpectedNoLintLinter:\"\"}: no file path for issue"
  level=warning msg="[runner] Can't process result by nolint processor: can't filter issue result.Issue{FromLinter:\"contextcheck\", Text:\"Function `Start$1` should pass the context parameter\", Severity:\"\", SourceLines:[]string(nil), Replacement:(*result.Replacement)(nil), Pkg:(*packages.Package)(0xc0014d6480), LineRange:(*result.Range)(nil), Pos:token.Position{Filename:\"\", Offset:0, Line:0, Column:0}, HunkPos:0, ExpectNoLint:false, ExpectedNoLintLinter:\"\"}: no file path for issue"
  level=warning msg="[runner/source_code] Failed to get line 0 for file : failed to get file  lines cache: can't get file  bytes from cache: can't read file : open : no such file or directory"
```

This issue started yesterday and with the previous version 1.56.2 it was working.
Latest version of contextcheck is from 1 year ago so I suppose this is a problem related to some change in the latest version.

I tried to reproduce this issue locally but I haven't had any success.
I write this issue anyway so maybe someone can discover what it is.
We disabled this specific linter completely in our pipeline as a temporary fix.

### Version of golangci-lint

1.57.0

This is the Github Actions pipeline step:
```
 - name: Lint code
        uses: golangci/golangci-lint-action@v4
        with:
          version: latest
          skip-cache: true
        env:
          GOFLAGS: "-buildvcs=false"
```

### Configuration
```yml
run:
  timeout: 2m
  modules-download-mode: readonly

linters:
  enable:
    - gosimple
    - govet
    - ineffassign
    - staticcheck
    - unused
    - asasalint
    - asciicheck
    - bidichk
    - bodyclose
    - containedctx
    - contextcheck
    - decorder
    - dogsled
    - dupl
    - dupword
    - durationcheck
    - errchkjson
    - errorlint
    - errname
    - execinquery
    - exportloopref
    - forbidigo
    - forcetypeassert
    - gci
    - goconst
    - gocritic
    - godot
    - gofmt
    - gofumpt
    - gomodguard
    - goprintffuncname
    - gosec
    - gosmopolitan
    - grouper
    - importas
    - interfacebloat
    - lll
    - loggercheck
    - makezero
    - mirror
    - misspell
    - nakedret
    - nestif
    - nilerr
    - nilnil
    - noctx
    - nolintlint
    - predeclared
    - reassign
    - revive
    - sloglint
    - sqlclosecheck
    - tagalign
    - tenv
    - testableexamples
    - testifylint
    - thelper
    - tparallel
    - unconvert
    - unparam
    - usestdlibvars
    - wastedassign
    - whitespace

  disable:
    - errcheck
    - cyclop
    - deadcode
    - depguard
    - exhaustive
    - exhaustivestruct
    - exhaustruct
    - funlen
    - ginkgolinter
    - gocognit
    - gocyclo
    - godox
    - goheader
    - goimports
    - goerr113
    - golint
    - gomnd
    - gomoddirectives
    - ifshort
    - inamedparam
    - interfacer
    - ireturn
    - maintidx
    - maligned
    - musttag
    - nlreturn
    - nonamedreturns
    - nosnakecase
    - nosprintfhostport
    - paralleltest
    - perfsprint
    - prealloc
    - promlinter
    - protogetter
    - rowserrcheck
    - scopelint
    - spancheck
    - structcheck
    - stylecheck
    - tagliatelle
    - testpackage
    - varcheck
    - varnamelen
    - wrapcheck
    - zerologlint
    - wsl
```

### Go environment

We use GitHub actions with this configuration step:
```
- name: Setup Go
        uses: actions/setup-go@v5
        with:
          go-version: 1.22
          cache: false
```

### Verbose output of running

-

### A minimal reproducible example or link to a public repository

-

### Validation

- [X] Yes, I've included all information above (version, config, etc.).

**Comment by ldez** [comment]
duplicate of #4550
